cmd/ctr/commands/client.go

```diff
@@ -62,5 +65,22 @@ func NewClient(context *cli.Context, opts ...containerd.Opt) (*containerd.Client
 		return nil, nil, nil, err
 	}
 	ctx, cancel := AppContext(context)
+	var suppressDeprecationWarnings bool
+	if s := os.Getenv("CONTAINERD_SUPPRESS_DEPRECATION_WARNINGS"); s != "" {
```

Comment:
Can we use `os.LookupEnv`
```suggestion
	if s, ok := os.LookupEnv("CONTAINERD_SUPPRESS_DEPRECATION_WARNINGS"); ok {
```

Reply:
I don't see benefit

Reply:
Thinking over it again. Since we need to ignore both unset and empty string cases, I think the current method is good. can ignore my comment.